pkg/controller/direct/monitoring/roundtrip_test.go

```diff
@@ -58,33 +57,17 @@ func FuzzMonitoringDashboardSpec(f *testing.F) {
 			".row_layout.rows[].widgets[]",
 		}
 		for _, widgetPath := range widgetPaths {
-			unimplementedFields.Insert(widgetPath + ".xy_chart.data_sets[].target_axis")
-			unimplementedFields.Insert(widgetPath + ".xy_chart.data_sets[].time_series_query.prometheus_query")
-			unimplementedFields.Insert(widgetPath + ".xy_chart.y2_axis")
-			unimplementedFields.Insert(widgetPath + ".xy_chart.timeshift_duration")
-			unimplementedFields.Insert(widgetPath + ".xy_chart.thresholds[].target_axis")
-
-			// This one might be a bug?
-			unimplementedFields.Insert(widgetPath + ".xy_chart.data_sets[].min_alignment_period.nanos")
-
-			unimplementedFields.Insert(widgetPath + ".scorecard.thresholds[].target_axis")
 			unimplementedFields.Insert(widgetPath + ".scorecard.blank_view")
```

Comment:
Blank view I think we probably should support, but I think it can go into 1.121 if need be.